src/main/java/org/example/leetcode/MaxStackSum.java

```diff
@@ -0,0 +1,45 @@
+package org.example.leetcode
+public class MaxStackSum {
+    public static void main(String[] args) {
+        int [] S1 = {4,2,3};
```

Comment:
Rename variables to follow camelCase (first letter small)

Eg. s1, n1, n2, etc.